include/zephyr/bluetooth/gatt.h

```diff
@@ -72,6 +72,18 @@ enum {
 	 *  passed to write callback.
 	 */
 	BT_GATT_PERM_PREPARE_WRITE = BIT(6),
+
+	/** @brief Attribute read permission with LESC based connection.
+	 *
+	 *  If set, requires that LE Secure Connections is used for read access.
+	 */
+	BT_GATT_PERM_READ_LESC = BIT(7),
+
+	/** @brief Attribute write permission with LESC based connection.
```

Comment:
```suggestion
	/** @brief Attribute write permission with LE Secure Connection encryption.
```

Suggest to not use the LESC abbreviate in the brief, and to the same word pattern as the other perms